src/Build.UnitTests/BackEnd/LoggingServicesLogMethod_Tests.cs

```diff
@@ -1772,8 +1772,7 @@ public static IBuildComponent CreateLoggingService(LoggerMode mode, int nodeId,
             }
 
             /// <summary>
-            /// Override the method to log which event was p1446
-            /// rocessed so it can be verified in a test
+            /// Override the method to log which event was processed so it can be verified in a test
```

Comment:
Yeah, I wasn't sure if `p1446` was a code, but eventually say that it was an odd split of the word `processed`